components/General/Faram/computer.js

```diff
@@ -19,7 +20,15 @@ const computeOutputSettings = (obj, schema, dataList = []) => {
             let subSettings;
 
             if (isComputer) {
-                subSettings = { $set: childSchema(...newDataList) };
+                // Update the computed value only if it is different than existing value.
+                // This is to prevent immutability-helper from creating completely
+                // new object even when computed value is not changed.
+                const newData = childSchema(...newDataList);
+                const oldData = (obj || emptyObject)[fieldName];
+
+                subSettings = isEqual(oldData, newData)
```

Comment:
Are we only checking the reference for objects here? Is that the intended behavior?

Reply:
yes, that's exactly what we need here